torch/onnx/utils.py

```diff
@@ -183,6 +184,7 @@ def exporter_context(model, mode, verbose):
         yield (mode_ctx, apex_ctx, log_ctx)
 
 
+@_beartype.beartype
```

Comment:
Is there a way to apply to all functions/methods under a certain module?

Reply:
There should be. I saw a prototype in beartype's repo https://github.com/beartype/beartype/issues/114